src/main/java/sample/JWKSetEndpoint.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sample;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import java.io.IOException;
+import java.io.Writer;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import com.nimbusds.jose.jwk.JWKSet;
+
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class JWKSetEndpoint extends OncePerRequestFilter {
+
+    static final String WELL_KNOWN_JWK_URIS = "/.well-known/jwk_uris";
```

Comment:
wouldn't it be better if the uri was configurable and the constant especially accessbile?

Reply:
Thanks for the feedback @Captain-P-Goldfish. The goal is to start with the minimal amount of code to get things working. We will add custom configurations as we build things out.